components/style/values/specified/grid.rs

```diff
@@ -367,3 +367,48 @@ pub fn parse_line_names(input: &mut Parser) -> Result<Vec<String>, ()> {
         Ok(values)
     })
 }
+
+/// The initial argument of the `repeat` function.
+///
+/// https://drafts.csswg.org/css-grid/#typedef-track-repeat
+#[derive(Clone, Copy, PartialEq, Debug)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+pub enum RepeatCount {
+    /// A positive integer. This is allowed only for `<track-repeat>` and `<fixed-repeat>`
+    Number(u16),
```

Comment:
You're right. The funny thing is that gecko doesn't support it - https://bugzilla.mozilla.org/show_bug.cgi?format=default&id=1350069 😄